bin/sentieon-genomics-202010.02/doc/licsrvr.sh

```
@@ -0,0 +1,70 @@
+[ -f /etc/sysconfig/licsrvr ] && . /etc/sysconfig/licsrvr
+[ -f /etc/default/licsrvr ] && . /etc/default/licsrvr
+
+RETVAL=0
+prog="licsrvr"
+
+licsrvr=${licsrvr:-/usr/sbin/licsrvr}
+licfile=${licfile:-/etc/sentieon.lic}
+logfile=${logfile:-/var/log/licsrvr.log}
+
+start() {
+	echo -n "Starting $prog:"
```

Shellcheck (0.10.0)

[warning] In POSIX sh, echo flags are undefined.

(SC3037)